python/tvm/relay/frontend/tensorflow.py

```diff
@@ -784,6 +788,18 @@ def _impl(inputs, attr, params):
             size = _get_list_param(params, inputs[2])
         except (IndexError, KeyError, AttributeError):
             size = _infer_value(inputs[2], params).asnumpy().tolist()[0]
+        
+        # handle situations when return empty constant
+        input0_np = _infer_value_simulated(inputs[0], params).asnumpy()  
+        input0_shape = input0_np.shape
+        for input_dim_i, begin_i, size_i in zip(input0_shape, begin, size):
+            # when one size_i == 0, return empty constant
+            if size_i == 0:
+                return _expr.const([], dtype=input0_np.dtype) 
+            # when begin index larger than the dimension of inputs[0]
+            if begin_i >= input_dim_i:
+                return _expr.const([], dtype=input0_np.dtype) 
```

Comment:
I think strided_slice has already supported empty slice: https://github.com/apache/incubator-tvm/commit/8e2f229a12f5867e17f6328f57a2c86c8bc31ca1#diff-6bbbb63b8fa5dc215443d88cfafed27aL559

Reply:
You are right. Probably I tested on a outdated copy of the TVM. I will close this PR and thanks.